cinema/views.py

```diff
@@ -34,6 +37,33 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_ints(qs):
+        return [int(str_id) for str_id in qs.split(",")]
```

Comment:
Use duplicating this function in another Viewset. Reuse it

Reply:
not fixed